text/trace/0107-sampling-score.md

```diff
@@ -0,0 +1,217 @@
+# Associating sampling score with the trace
+
+Enable consistent sampling across distributed application with different
+sampling rates and probability calculation algorithms.
+
+## TL;DR
+
+**Score** is a floating point number associated with
+the trace. It's calculated when trace starts and flows in the `tracestate`,
+it's used by samplers to make consistent sampling decisions.
+
+Service that starts the trace calculates the score and adds it to the
+`tracestate` so downstream services can re-use it to make their sampling
+decisions *instead of* re-calculating score as a function of trace-id
+(or trace-flags).
+
+*Score* is not related to sampling *rate* (aka *probability* which represents
+sampler's configuration not specific to trace).
+
+## Motivation
+
+The goal is to enable a mechanism for consistent (best effort) sampling
+between services with different sampling rates and different probability
+calculation algorithms (for interoperability with existing tracing tools).
```

Comment:
You are referring couple of times that the trace id based algorithm does not work well enough, can you add in the motivation section the reason why?